src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+
+/**
+ * A normalizer that normalizes a ConstraintViolationListInterface instance.
+ *
+ * This Normalizer implements partially RFC7807 {@link https://tools.ietf.org/html/rfc7807}.
```

Comment:
I would say `implements RFC7807 partially.`

But can we be a bit more precise? Can we list what we do not support or how we diverge from the RFC? I think that would help our users.

Reply:
Actually I re-read the RFC and came up to the same conclusion as @teohhanhui's [comment](https://github.com/symfony/symfony/pull/22150#issuecomment-291249290).

We are allowed to omit the `type` property.

That's why I removed the "partially" because the normalizer do implement the RFC.